src/aurora/lua/scriptman.cpp

```diff
@@ -83,7 +85,7 @@ void ScriptManager::executeFile(const Common::UString &path) {
 
 	const int execResult = lua_dobuffer(_luaState, data, dataSize, path.c_str());
 	if (execResult != 0) {
-		warning("Failed to execute Lua file: %s", TypeMan.setFileType(path, kFileTypeLUC).c_str());
+		error("Failed to execute Lua file: %s", TypeMan.setFileType(path, kFileTypeLUC).c_str());
```

Comment:
Hmm, not sure I like having that an error(), which basically immediately kills xoreos. Maybe have it throw an exception, so that it can still be caught by the Witcher engine? That way, we can try to salvage the situation.

(The same for executeString())